src/main/kotlin/org/rust/cargo/project/model/impl/CargoProjectImpl.kt

```diff
@@ -227,7 +229,7 @@ open class CargoProjectsServiceImpl(
         project.modules
             .asSequence()
             .flatMap { ModuleRootManager.getInstance(it).contentRoots.asSequence() }
-            .mapNotNull { it.findChild(CargoConstants.MANIFEST_FILE) }
+            .mapNotNull { it.findChild("bazel-bin")?.findChild(CargoConstants.MANIFEST_FILE) ?: it.findChild(CargoConstants.MANIFEST_FILE) }
```

Comment:
The `BUILD` files of a Bazel project contain build "targets" including Rust binaries and libraries. Critically, these targets contain all of the information needed to run or import a crate, eliminating the need for a Cargo manifest, `Cargo.toml`.

Given that the architecture of `intellij-rust` is based on Cargo, we have adopted the strategy of using Bazel to generate `Cargo.toml` files, which it does in its sandbox, accessible via the symlink `{projectRoot}/bazel-bin`.